### PR TITLE
Fixed logic for document errors. 1.7.0

### PR DESCRIPTION
**Pull request by jleonardw9**

### CODE REVIEW

File: app/controllers/service_requests_controller.rb [hunk 1]
```diff
@@ -574,7 +574,7 @@ def document_save_update errors
     doc_type_other = params[:doc_type_other]
     upload_clicked = params[:upload_clicked]
 
-    if doc_type and process_ssr_organization_ids and (document or document_id)
+    if !doc_type.empty? && process_ssr_organization_ids && document
```
Comment (wtholt) [inline app/controllers/service_requests_controller.rb]: Could we say

    doc_type.any?

Comment (jleonardw9) [inline app/controllers/service_requests_controller.rb]: Good point. Making the change.

Comment (kyle-glick) [inline app/controllers/service_requests_controller.rb]: Can't call .any? on Strings

Comment (jleonardw9) [inline app/controllers/service_requests_controller.rb]: Right, and doc_type will either be an empty string or a string like 'Protocol', etc.

**Pull request merged**